src/dts.ts

```diff
@@ -1,3 +1,3 @@
   map(projection: (it: T) => any): Queryable<any>;\n        map(projection: string): Queryable<any>;\n        orderBy(predicate: (it: T) => void): Queryable<T>;\n        orderBy(predicate: string): Queryable<T>;\n        orderByDescending(predicate: (it: T) => void): Queryable<T>;\n        orderByDescending(predicate: string): Queryable<T>;\n        include(selector: string): Queryable<T>;\n        skip(amount: number): Queryable<T>;\n        take(amount: number): Queryable<T>;\n        forEach(handler: (it: T) => void): Promise<T>;\n        length(): Promise<number>;\n        toArray(): Promise<T[]>;\n        single(predicate: (it: T) => boolean, params?: any): Promise<T>;\n        single(predicate: string, params?: any): Promise<T>;\n        first(predicate?: (it: T) => boolean, params?: any): Promise<T>;\n        first(predicate?: string, params?: any): Promise<T>;\n        removeAll(): Promise<number>;\n    }\n    class ServiceAction{}\n    class ServiceFunction{}\n\n    function implementation(name:string): typeof Base;\n}\nexport {$data as $data}\n\ndeclare type JSDate = Date;\n\ndeclare module Edm {\n    type Boolean = boolean;\n    type Binary = Uint8Array;\n    type DateTime = JSDate;\n    type DateTimeOffset = JSDate;\n    type Duration = string;\n    type TimeOfDay = string;\n    type Date = string;\n    type Time = string;\n    type Decimal = string;\n    type Single = number;\n    type Float = number;\n    type Double = number;\n    type Guid = string;\n    type Int16 = number;\n    type Int32 = number;\n    type Int64 = string;\n    type Byte = number;\n    type SByte = number;\n    type String = string;\n    type GeographyPoint = $data.Geography;\n    type GeographyLineString = $data.GeographyLineString;\n    type GeographyPolygon = $data.GeographyPolygon;\n    type GeographyMultiPoint = $data.GeographyMultiPoint;\n    type GeographyMultiPolygon = $data.GeographyMultiPolygon;\n    type GeographyMultiLineString = $data.GeographyMultiLineString;\n    type GeographyCollection = $data.GeographyCollection;\n    type GeometryPoint = $data.Geometry;\n    type GeometryLineString = $data.GeometryLineString;\n    type GeometryPolygon = $data.GeometryPolygon;\n    type GeometryMultiPoint = $data.GeometryMultiPoint;\n    type GeometryMultiPolygon = $data.GeometryMultiPolygon;\n    type GeometryMultiLineString = $data.GeometryMultiLineString;\n    type GeometryCollection = $data.GeometryCollection;\n    type Primitive =\n        Boolean | Binary | Guid | DateTime | DateTimeOffset | Duration | TimeOfDay | Date | Time |\n        Decimal | Single | Float | Double | Int16 | Int32 | Int64 | Byte | SByte | String |\n        GeographyPoint | GeographyLineString | GeographyPolygon | GeographyMultiPoint | GeographyMultiLineString | GeographyMultiPolygon | GeographyCollection |\n        GeometryPoint | GeometryLineString | GeometryPolygon | GeometryMultiPoint | GeometryMultiLineString | GeometryMultiPolygon | GeometryCollection;\n}\nexport {Edm as Edm}"
```

Comment:
added Queryable.count() in here, which is hard to spot via the GitHub diff interface.